.idea/inspectionProfiles/Druid.xml

```diff
@@ -311,6 +311,11 @@
         <constraint name="k" within="" contains="" />
         <constraint name="v" within="" contains="" />
       </searchConfiguration>
+      <searchConfiguration name="Prohibit Thread.getState() != TERMINATED antipattern" text="while($k$.getDelayedCloseExecutor().getState()!=Thread.State.$k1$){Thread.sleep($k2$)}" recursive="true" caseInsensitive="true" type="JAVA">
```

Comment:
`getDelayedCloseExecutor()` is still too specific. I think the pattern should just be
```
$t$.getState() == Thread.State.$state$
```
Where the type of `$t$` is filtered as `java.lang.Thread`.
Another pattern is
```
$t$.getState() != Thread.State.$state$
```